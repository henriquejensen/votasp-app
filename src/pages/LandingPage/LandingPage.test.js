import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import LandingPage from './LandingPage';

describe('LandingPage', () => {
  it('should match a snapshot', () => {
    expect(toJson(shallow(( <LandingPage /> )))).toMatchSnapshot();
  })
})
