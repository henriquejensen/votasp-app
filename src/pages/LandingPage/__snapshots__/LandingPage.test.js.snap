// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage should match a snapshot 1`] = `
<PageLayout>
  <div
    className="landing-page__projeto"
  >
    <div
      className="landing-page__descricao container"
    >
      <h1>
        O projeto
      </h1>
      <p>
        O Vota SP é um site que vai ajudar a população do Estado de São Paulo escolher seus candidatos e candidatas a deputado, deputada estadual e federal.
      </p>
      <p>
        Através de questões objetivas os eleitores podem verificar sua afinidade com os candidatos e candidatas. 
        <strong>
          É muito simples e prático!
        </strong>
      </p>
    </div>
    <div
      className="como-funciona"
    >
      <Component />
    </div>
    <div
      className="landing-page__call-to-action"
    >
      <NavLink
        activeClassName="active"
        aria-current="page"
        className="btn btn-primary btn-lg"
        to="/app/questionario/1"
      >
        Quero participar!
      </NavLink>
    </div>
  </div>
  <div
    className="landing-page__realizacao"
  >
    <div
      className="landing-page__descricao container"
    >
      <Realizacao />
    </div>
  </div>
</PageLayout>
`;
